front/app/containers/Admin/projects/project/projectHeader/index.tsx

```diff
@@ -171,6 +172,57 @@ const ProjectHeader = ({ projectId }: Props) => {
           <Text color="coolGrey600" fontSize="s" my="0px">
             ·
           </Text>
+          <Tooltip
+            theme="dark"
+            maxWidth={280}
+            placement="bottom"
+            content={
+              <Box p="8px">
+                <FormattedMessage {...messages.participantsInfoTitle} />
+                <ul
+                  style={{
+                    margin: '0',
+                    marginBottom: '8px',
+                    listStyleType: 'disc',
+                  }}
+                >
+                  <li>
+                    <FormattedMessage {...messages.users} />
+                  </li>
+                  <li>
+                    <FormattedMessage {...messages.attendees} />
+                  </li>
+                </ul>
+
+                <FormattedMessage {...messages.participantsExclusionTitle} />
+                <ul
+                  style={{
+                    margin: '0',
+                    marginBottom: '8px',
+                    listStyleType: 'disc',
+                  }}
+                >
+                  <li>
+                    <FormattedMessage {...messages.followers} />
+                  </li>
+                  <li>
+                    <FormattedMessage {...messages.embeddedMethods} />
+                  </li>
+                </ul>
+
+                <FormattedMessage {...messages.note} />
+              </Box>
+            }
+          >
+            <Box display="flex" alignItems="center">
+              <Icon name="user" fill={colors.coolGrey600} width="16px" />
+              <Text color="coolGrey600" fontSize="s" m="0px">
+                {formatMessage(messages.participants, {
+                  participantsCount: project.data.attributes.participants_count,
+                })}
+              </Text>
+            </Box>
+          </Tooltip>
```

Comment:
@EdwinKato I think you can remove the "participants count" message below then correct?? It looks like we're showing it twice here:
![image](https://github.com/user-attachments/assets/313a85bc-339f-4349-ac3f-ee7e5b2b3932)